src/app/core/auth/auth.service.ts

```diff
@@ -337,8 +348,12 @@ export class AuthService {
           this.router.navigated = false;
           const url = decodeURIComponent(redirectUrl);
           this.router.navigateByUrl(url);
+          /* TODO Reenable hard redirect when REST API can handle x-forwarded-for */
+          // this._window.nativeWindow.location.href = url;
         } else {
           this.router.navigate(['/']);
+          /* TODO Reenable hard redirect when REST API can handle x-forwarded-for */
```

Comment:
Thanks for the TODO.  Do we have a REST API bug ticket that this should/could refer to?  I'd like to ensure this gets logged (or has been logged) as a REST API bug.

Ideally, these sorts of TODOs (which reference a known bug) should mention the bug ticket, e.g. DS-100, so that we can easily know if the bug is fixed.

Reply:
This REST PR should fix this particular issue: https://github.com/DSpace/DSpace/pull/2207